### ISSUE TITLE
Override Keyboard Navigation

### ISSUE THREAD

**Issue by arqlz** [MicrosoftDocs/windows-uwp#1407]
How to override Keyboard Navigation so none action is done when UP, DOWN, LEFT and RIGHT keys are pressed? 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d5ba995b-8bf3-407c-7b87-015d22d813f1
* Version Independent ID: 52c553b6-e94d-2810-4123-8ee215b002bf
* Content: [Tree view - Windows UWP applications](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/tree-view)
* Content Source: [windows-apps-src/design/controls-and-patterns/tree-view.md](https://github.com/MicrosoftDocs/windows-uwp/blob/docs/windows-apps-src/design/controls-and-patterns/tree-view.md)
* Product: **uwp**
* GitHub Login: @jwmsft
* Microsoft Alias: **jimwalk**

**Comment by ghost** [comment]
In the treeView create event PreviewKeyDown="TreeView_PreviewKeyDown"

        private void TreeView_PreviewKeyDown(object sender, KeyRoutedEventArgs e)
        {
            e.Handled = true;
        }

This will do the trick.

**Comment by arqlz** [comment]
No, if you do that none other component will receive the keypress event... I need to handle it on a sub element.  

example

```
 Treeview
        TreeViewItem
                TextBox <-- here i want to handle the event
```

**Comment by GrantMeStrength** [comment]
Closing this issue due to age (filed before 2024). If this is still relevant with current Windows App SDK / WinUI 3 documentation, please reopen this issue or file a new one with updated details. Thank you for your contribution!
